terraform/scripts/worker_bootstrap.sh

```diff
@@ -6,6 +6,8 @@ set -eux
 # steps with the hab package
 sudo apt-get update
 sudo apt-get -y install docker.io
+sudo systemctl start docker
```

Comment:
Hm. What would be the benefit?

Reply:
Reduced complexity.  Rather than start, sleep, stop, start,  it feels like there would be less likelihood of something going sideways if it was possible to have the filesystem in the desired state before we installed and started the package.